quantus/helpers/explanation_func_tf.py

```diff
@@ -0,0 +1,82 @@
+"""This modules contains explainer functions which can be used in conjunction with the metrics in the library."""
+from typing import Union
+
+import numpy as np
+import scipy
+import random
+import cv2
+from tf_explain.core.vanilla_gradients import VanillaGradients
+import warnings
+from .utils import *
+from .normalise_func import *
+import tensorflow as tf
+
+
+def explain_tf(
+    model: tf.keras.Model,
+    inputs: Union[np.array, torch.Tensor],
+    targets: Union[np.array, torch.Tensor],
+    *args,
+    **kwargs,
+) -> np.ndarray:
+    """
+    Explain inputs given a model, targets and an explanation method.
+
+    Expecting inputs to be shaped such as (batch_size, img_size, img_size, nr_channels)
+
+    Returns np.ndarray of same shape as inputs.
+    """
+
+    if "method" not in kwargs:
+        warnings.warn(
+            f"Using quantus 'explain' function as an explainer without specifying 'method' (str) "
+            f"in kwargs will produce a vanilla 'Gradient' explanation.\n",
+            category=UserWarning,
+        )
+
+    method = kwargs.get("method", "Gradient").lower()
+
+    inputs = inputs.reshape(
```

Comment:
why not use make use of model.shape_input here?

Reply:
@dilyabareeva

Reply:
we pass a tf.keras.Model object to this function, not a PyTorchModel, so we can't call this method

Reply:
true! however, this restricts explanations to models with channels_last ordering inputs. Since we have the model interface class, I think it makes sense to try and support flexibility where possible (as we should make as few assumptions about the model as possible to improve usability)

So maybe the passed input could be an instance of TensorFlowModel instead of a tf.keras.Model? Or is there a reason not to?